Mathlib/Algebra/Category/FiniteGrp/Basic.lean

```diff
@@ -0,0 +1,85 @@
+/-
+Copyright (c) 2024 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang, Nailin Guan, Yuyang Zhao
+-/
+import Mathlib.Algebra.Category.Grp.Basic
+
+/-!
+
+## Main definitions and results
+
+* `FiniteGrp` is the type of finite groups.
```

Comment:
Probably "the category of finite groups", no?

Reply:
sorry that I forgot to change this (The profinite one is correct)

Reply:
fixed